src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_VALUE = 100;
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random randomValueSupplier = new Random();
+
+    public Ball getRandomBall() {
+        int number = randomValueSupplier.nextInt(MAX_VALUE);
```

Comment:
You've changed MAX_VALUE correctly, but now in nextInt() you have add 1, so it will randomize a number between 0 and 100 inclusively. Here are the docs: https://docs.oracle.com/javase/8/docs/api/java/util/Random.html#nextInt-int-
Your code should look like:
`int number = randomValueSupplier.nextInt(MAX_VALUE + 1);`